source/_reference-t-z.html.md.erb

```diff
@@ -73,11 +134,13 @@ Gets an access token, as part of the OAuth process. For more information, see [O
 
      |     |
 -----|-----|
+<%= HEADERS %> | **Content-Type**: application/x-www-form-urlencoded
 <%= PARAMETERS %> | **grant_type** (required) -- must be set to "authorization_code"
  | **client_id** (required) -- client id for your app
- | **code** (required) -- authorization code returned in the previous step
- | **redirect_uri** (optional) -- redirect URL registered for your app, including the protocol (for example, "https://"); if not provided, the redirect URL set during registration is used.
- | **hash** (required) -- SHA-256 hash of your app secret concatenated with a pipe and the authorization code.  The app_secret is never sent with the request.
+ | **code** (required) -- authorization code returned during the OAuth Flow.
+ | **redirect_uri** (deprecated) -- redirect URL registered for your app, including the protocol (for example, "https://"); if not provided, the redirect URL set during registration is used.
```

Comment:
redirect_uri is listed as deprecated here, but optional below.

Reply:
Fixed.